### DISCUSSION TITLE
Testing Idena node: release candidate 0.27.3-rc1

### DISCUSSION THREAD

**Discussion by midenaio** [idena-network/idena-go#835] [General]
We invite you to test the release candidate version of the Idena node: 0.27-3-rc1.
Please [download](https://github.com/idena-network/idena-go/releases/tag/v0.27.3-rc1) the node manually if you want to join the testing.

Changes:

- Use the last hard fork genesis block for sync: #3417451
- Performance improvements 
  - Use batches for gossip
  - Disable generation of old versions of snapshots 
- Validation improvements:
  - Broadcast open short answers 2 minutes after the start of validation
  - Broadcast open short answers after successful long answers broadcast if needed
  - Add client type info into open short session answers 
  - Respect flip `keyPackages` in evidence map

We plan to release these changes in two days unless negative feedback is received

**Comment by rioda-org** [comment]
My experience was 0.27-3-rc1 version can't stay connected to peers as good as 0.27.2 does
1-3 peers oposed to 6-7 peers for 0.27.2

This is in-home testing, behind NAT and CGNAT :)
